stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -32,61 +31,13 @@ protected void onCopyAttributes(View element, AttributeAccumulator attributes) {
     }
   }
 
-  private static int getResourcePackageId(int id) {
-    return (id >>> 24) & 0xff;
-  }
-
   @Nullable
   private static String getIdAttribute(View element) {
-    // Adapted from View.toString()
-
     int id = element.getId();
     if (id == View.NO_ID) {
```

Comment:
why don't we make it so that getIdStringQuietly() handles the 'id == NO_ID' case?